circle.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       - CC:  gcc-6
       - GCOV: gcov-6
       - GENERATOR: Ninja
-      - BUILD_PARALLEL_JOBS: 3
+      - BUILD_PARALLEL_JOBS: 8
```

Comment:
Oh, sorry. I think it was 3 before LOG changes. You can increase it to 4 to check, but that's not required if it was not decreased by LOG changes.

Reply:
Ok (i think it was decreased in the beginning but then I rebased and kept the value from `develop`)

Reply:
GCC cannot handle 8 for sure, mostly because it also produces debug symbols.

Reply:
It's back to 3 now